lib/legato/management/web_property.rb

```diff
@@ -36,18 +36,41 @@ def self.for_account(account)
       end
 
       def profiles
-        Profile.for_web_property(self)
+        @profiles ||= Profile.for_web_property(self)
+      end
+
+      def profiles=(value)
+        @profiles = value
       end
 
       def account
         @account ||= Account.from_child(self)
       end
 
+      def account=(value)
+        @account = value
+        @account_id = value.id
+      end
+
       def self.from_child(child)
         path = new({:id => child.web_property_id, :account_id => child.account_id}, nil).path
 
         get(child.user, path)
       end
+
+      def self.build_from_summary(attributes, user, account)
+        profiles = attributes['profiles'] || attributes[:profiles]
+        attributes.delete('profiles')
```

Comment:
What if the attributes uses symbol `:profiles`. You can probably combine these two lines like:

```ruby
profiles = attributes.delete('profiles') || attributes.delete(:profiles)
```